frappe/public/js/libs.bundle.js

```diff
@@ -1,5 +1,6 @@
 import "./jquery-bootstrap";
 import "./lib/moment";
+import hljs from "./syntax_highlight";
```

Comment:
Why is this loaded in `libs.bundle.js` which only loads in desk where it's not used?

Reply:
It was previously part of the closure, albeit included in website. I had little idea where this is actually used, in practice. It's a "lib" so `libs.bundle` did nothing more than "sound about right".

As a lib, having its cost on size, I felt that leaving it in website would otherwise defy the purpose of this PR.

From the realization that you prompt: should this included in a website variant?

Reply:
To solve the combinatory problem, we could also add a specialized router that can assemple such a resource declaration on the fly:

`/assets/.../website.realtime.higlight.[...].js` out of:

```
website.js
website-realtime.js
website-highlight.js
[...]
```

Wdyt?

Reply:
Main users are `/app` at present, I'd optimize for that over web any day :sweat_smile:

Reply:
Why not just keep it in web bundle like before? even right now it's being included indirectly but now it's ALSO being included in `/app` bundles?

Reply:
Oh yah. That makes absolutely (no) sense. :smile:

Reply:
@ankush f67d3d3a2bf45a2f510b7bb7d55fe0791309c15a should address this, finally.